packages/paste-core/components/meter/src/index.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import {type BoxProps, Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {ScreenReaderOnly} from '@twilio-paste/screen-reader-only';
+import type {HTMLPasteProps} from '@twilio-paste/types';
+
+export interface MeterProps extends HTMLPasteProps<'meter'>, Pick<BoxProps, 'element'> {
+  children: NonNullable<React.ReactNode>;
+  ariaLabelledby: string;
+  min?: number;
+  low?: number;
+  max?: number;
+  high?: number;
+  value?: number;
+  optimum?: number;
+}
+
+const Meter = React.forwardRef<HTMLMeterElement, MeterProps>(
+  ({element = 'METER', ariaLabelledby, children, min = 0, max = 1, value = 0, ...props}, ref) => {
+    const meterWidthInPixels = 296; // size30
```

Comment:
todo: get from useTheme or just make this 100% width and set the barWidth to a % value